connman/src/service.c

```diff
@@ -3470,6 +3491,14 @@ static void append_struct(gpointer value, gpointer user_data)
 	append_struct_service(iter, append_dict_properties, service);
 }
 
+static void append_dict_properties_updated(DBusMessageIter *dict,
+	void *user_data)
```

Comment:
[Coding style M4](https://github.com/sailfishos/connman/blob/master/connman/doc/coding-style.txt#L78) instructs to intend these new lines to the right.

Reply:
Thanks a lot for the review, I have fixed that

Reply:
Could you intend this to the right up to the 80 char limit in style of, e.g. [append_ext_wifi_info()](https://github.com/sailfishos/connman/blob/master/connman/src/service.c#L2994) ?

I know the code has in itself different styles but each new functionality we add should at least try to follow the current coding styles. My point here is that at some point we'd like to upstream as much as possible we have and when we do the style changes at the steps of adding new code we probably avoid one additional review round in the future :)

Reply:
Thanks a lot. I have created tabs like in surrounded code, but it seems that it was created so long time ago :)